cards/src/main/resources/cards/custom/group3/bug/verdant_dreams/spell_dream_of_kinship.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "Dream of Kinship",
+  "baseManaCost": 10,
+  "type": "SPELL",
+  "heroClass": "AMBER",
+  "rarity": "LEGENDARY",
+  "description": "Discover a Beast, a Dragon, an Elemental, and a Fae. Summon them all.",
+  "targetSelection": "NONE",
+  "spell": {
+    "class": "MetaSpell",
+    "spells": [
+      {
+        "class": "DiscoverSpell",
+        "spell": {
+          "class": "SummonSpell"
+        },
+        "cardFilter": {
+          "class": "CardFilter",
+          "race": "BEAST"
```

Comment:
these might need `"cardSource": {"class": "UnweightedCatalogueSource"}`, are they only supposed to discover beast, dragon, elemental and fae AMBER cards or among all?

Reply:
Phyley says it's supposed to only be from AMBER/Neutral and I think this does that, so we should be good.